source/index.html

```diff
@@ -1,12 +1,170 @@
+            <h3 class="products__title">Похудение</h3>
+            <p class="products__text">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора
+              на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.</p>
+            <a class="products__link link" href=""
+              aria-label="Перейти на страницу каталога продукции для похудения Cat Energy  Slim">Каталог slim</a>
+          </li>
+          <li class="products__item products__item--pro">
+            <h3 class="products__title">Набор массы</h3>
+            <p class="products__text">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro
+              поможет вашему коту нарастить необходимые мышцы!</p>
+            <a class="products__link link" href=""
+              aria-label="Перейти на страницу каталога продукции для набора массы Cat Energy  Pro">Каталог pro</a>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="info">
+      <div class="wrapper">
+        <h2 class="info__title  title">Как это работает</h2>
+        <ul class="info__list list">
+          <li class="info__item info__item--eco">
+            <p class="info__text">Функциональное питание содержит только полезные питательные вещества.</p>
+          <li class="info__item info__item--form">
+            <p class="info__text">Выпускается в виде порошка, который нужно лишь залить кипятком и готово.</p>
+          <li class="info__item info__item--use">
+            <p class="info__text">Замените один-два приема обычной еды на наше функциональное питание.</p>
+          <li class="info__item info__item--time">
+            <p class="info__text">Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!</p>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="example">
+      <div class="wrapper">
+        <h2 class="example__title title">Живой пример</h2>
+        <p class="example__text">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim.
+          Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16
+          часов в день. </p>
+        <dl class="example__list">
+          <div class="example__item">
+            <dt class="example__number">5 кг</dt>
+            <dd class="example__description">снижение веса</dd>
+          </div>
+          <div class="example__item">
+            <dt class="example__number">60 дней</dt>
+            <dd class="example__description">затрачено времени</dd>
+          </div>
+        </dl>
+        <p class="example__total">Затраты на питание: 15 000 РУБ.</p>
+        <div class="example__slider slider">
+          <div class="slider__content">
+            <img src="img/cat/tall-cat.jpg" width="311" height="232" alt="Кот с избыточным весом"
+              class="slider__img slider__img--active">
```

Comment:
```suggestion
              class="slider__img">
```

тут можно без мода (модификатора), тк картинки ничем не отличаются в зависимости от активности.

Reply:
я поставила для дальнейшей реализации слайдера, повесить на него display: block например, а без display: none

Reply:
Это называется "предварительная оптимизация". Лучше этот мод добавить именно тогда, когда ты будешь делать слайдер. Тут такая тонка грань, что тебе надо думать на будущее, при этом заранее многого не делать,  тк предварительная оптимизация - зло. Зло, потому что ты тратишь время на то, что может не пригодиться. Например, что, если ты решишь сделать слайдер на js?